src/github.com/getlantern/lantern-mobile/app/src/main/java/org/getlantern/lantern/fragment/FeedFragment.java

```diff
@@ -62,28 +55,48 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View view = inflater.inflate(R.layout.feed_fragment, container, false);
 
         mList = (ListView)view.findViewById(R.id.feed);
-        adapter = new FeedAdapter(getActivity(), mFeedItems);
         mList.setAdapter(adapter);
         return view;
     }
 
+    protected class LoadFeed extends AsyncTask<String, Void, List<FeedItem>> {
```

Comment:
Seems like a private class would make more sense here @atavism -- i.e. I don't think we're ever going to subclass this and nothing else in this package will user, or maybe more precisely I don't think we want to allow either of those to happen.